pandas/core/generic.py

```diff
@@ -12022,8 +12022,8 @@ def last_valid_index(self) -> Hashable:
 
 Returns
 -------
-{name1} or {name2}
-    If level is specified, then, {name2} is returned; otherwise, {name1}
+{name1} or scalar
+    If axis=None, then a scalar boolean is returned. Otherwise, a {name1} is returned with index matching the index argument.
```

Comment:
There shouldn't be a period at the end. See how it renders here: https://pandas.pydata.org/preview/pandas-dev/pandas/57817/docs/reference/api/pandas.Series.any.html#pandas.Series.any

Also, the type is `scalar or scalar`. I think this already happens in the docs, but since you are already changing this, maybe worth leaving it well.

Also note that while the discussion is not settle, people seems to be happy to stop reusing docstrings. So if you want to replicate the docstring for each method, that would be ok.